utilities/map.metta

```diff
@@ -91,14 +91,14 @@
 ;; Get the index of a key value pair given a key
 (: Map.find (-> (Map ($k $v)) $k Number))
 (= (Map.find NilMap $k) -1)
-(= (Map.find (ConsMap ($curKey $curVal) $xs) $k')
-   (if (== $curKey $k')
+(= (Map.find (ConsMap ($k $v) $xs) $k')
+   (if (== $k $k')
        0
-       (chain (Map.find $xs $k') $targetIdx
+       (chain (Map.find $xs $k'' ) $targetIdx
```

Comment:
I don't understand why the `$k'` is changed into `$k''` there's no such variable definition.


```suggestion
       (chain (Map.find $xs $k') $targetIdx
```